src/libYARP_OS/src/BottleImpl.cpp

```diff
@@ -43,6 +44,22 @@ using yarp::os::Value;
 
 #define YARP_STRINIT(len) ((size_t)(len)), 0
 
+/*
+ * The maximum string length for a 'double' printed as a string using ("%.*g", DBL_DIG) will be:
+ *  Initial +/- sign                        1 char
+ *  First digit for exponential notation    1 char
+ * '.' decimal separator char               1 char
+ *  DBL_DIG digits for the mantissa         DBL_DIG chars
+ * 'e+/-'                                   2 chars
+ * YARP_DBL_EXP_DIG  for the exponential    YARP_DBL_EXP_DIG chars
+ * string terminator                        1 char
+ * FILLER                                   10 chars  (you know, for safety)
+ * -----------------------------------------------------
+ * TOTAL is                                 16 + DBL_DIG + YARP_DBL_EXP_DIG
+ */
+#define YARP_DOUBLE_TO_STRING_MAX_LENGHT       16+ DBL_DIG +YARP_DBL_EXP_DIG
```

Comment:
small typo: lenght -> length

Reply:
🙃

Reply:
And...I know I will sound peaky but... could you please also add some spaces in the `#define`, just like it is written in `TOTAL is`.

Reply:
Ops, I did it again